app/src/main/java/org/eyeseetea/malariacare/SettingsActivity.java

```diff
@@ -916,6 +916,7 @@ void launchLoginOnEulaAccepted(){
     }
 
     Intent prepareIntent(){
+        ProgressActivity.PULL_CANCEL =true;
```

Comment:
@Strabagkantza correct me if I'm wrong but you're here avoiding the pull after the login. So now there's no pull. Can you explain this a little bit more? Maybe put a comment detailing your intentions?